fixtures/callbacks/src/lib.rs

```diff
@@ -33,6 +33,15 @@ impl RustGetters {
     fn get_list(&self, callback: Box<dyn ForeignGetters>, v: Vec<i32>, arg2: bool) -> Vec<i32> {
         callback.get_list(v, arg2)
     }
+
+    fn get_string_optional_callback<'a>(
```

Comment:
FWIW, this seemed to work correctly for me without the explicit `'a` lifetime annotation, is there a reason it is required?